ebpf/xdp_filter.c

```diff
@@ -65,65 +84,108 @@ struct flowv6_keys {
         __u16 port16[2];
     };
     __u32 ip_proto;
-} __attribute__((__aligned__(8)));
+    __u16 vlan_id[2];
+};
 
 struct pair {
-    __u64 time;
-    __u64 packets;
-    __u64 bytes;
-} __attribute__((__aligned__(8)));
+    __u32 packets;
+    __u32 bytes;
```

Comment:
I guess this has consequences for accuracy on high bw elephant flows. We sync these counters once every ~10 seconds AFAICS. This means that we'd be unable to accurately track flows faster than ~4Gbit/s? These would overflow within that window. Is this correct? Are there other side effects?